payment_manual/indico_payment_manual/plugin.py

```diff
@@ -26,30 +27,37 @@
 from indico.util.placeholders import render_placeholder_info, replace_placeholders
 from indico.web.forms.validators import UsedIf
 
+from indico_payment_manual.blueprint import blueprint
 from indico_payment_manual import _
 
 
 DETAILS_DESC = _('The details the user needs to make their payment. This usually includes the bank account details, '
                  'the IBAN and payment reference.  You can also link to a custom payment site. In this case make sure '
                  'to use the URL-escaped versions of the placeholders where available.')
+TRIGGER_DESC = _('If enabled, registrations are put into pending payment state if users check out using this payment '
+                 'method.')
 
 
 class PluginSettingsForm(PaymentPluginSettingsFormBase):
     details = TextAreaField(_('Payment details'), [])
+    trigger_pending = BooleanField(_('Trigger pending state'), [])
 
     def __init__(self, *args, **kwargs):
         super(PluginSettingsForm, self).__init__(*args, **kwargs)
         self.details.description = DETAILS_DESC + '\n' + render_placeholder_info('manual-payment-details',
                                                                                  regform=None, registration=None)
+        self.trigger_pending.description = TRIGGER_DESC
```

Comment:
Set this directly on the field (like it's done in most other forms). The only reason why we handle the description in here is because we add a dynamic part.